test/acceptance/features/steps/steps.py

```diff
@@ -249,7 +249,7 @@ def sbo_jq_is(context, jq_expression, sbr_name, json_value):
     openshift = Openshift()
     polling2.poll(lambda: json.loads(
         openshift.get_resource_info_by_jq("servicebinding", sbr_name, context.namespace.name, jq_expression,
-                                          wait=False)) == json_value, step=5, timeout=800,
+                                          wait=False)).startswith(json_value), step=5, timeout=800,
```

Comment:
this is going to result in a way more positive results - why is this change needed in this PR?

Reply:
I changed it to check the value of `.status.secret` when there are multiple Secret resources.
I will remove this change and create a separate step function for that requirement.

Reply:
Where there are multiple secret referred, then the binding secret will be generated as before, so no need to check anything new there.

Reply:
Yes, but there was no assertion for the `.status.secret` value which is dynamic in nature. Please take a look at the test that I added and let me know.

Reply:
in case when intermediate secret is generated and then injected, we should asset the following things:

* service binding Ready condition is true
* `.status.binding.name` is set, i.e. non empty
* the application sees injected env vars or files

The fact that intermediate secret name is made as `binding name` + `some hash` is an implementation detail and the user should not count on that - we can change that at any point in time.

Reply:
Done. 832c72fcec87d9f227ff5ea071ff759a112edc95